.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes**:
+Fixes #
+
+**Definition of Done:**
+- [ ] The code is sufficiently documented
+- [ ] The code is appropriately tested
```

Comment:
We maybe should add that the code was reviewed/merge or has a different person was made aware of. Currently, most of the time this knowledge is earlier lost since even when things getting documented, it's challenging to find it.

Reply:
I don't understand this suggestion. We have a pr workflow. If people actually take the time to review is another question.

Can you suggest into an item for this list that reflects the issue you're describing?

Reply:
We have the problem, as you have observed before, that we're often not doing a proper review. This item on the list should be remaining on not only having a 'LGTM' or '+1', but sharing the knowledge behind this with someone else. So it could be:

- [ ] Shared the changes with the Team. 

So we want to prevent having a low bus factor.

Reply:
thanks, added to the list